content/secrets-managment/_index.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
To reduce scope of this PR and unblock me, I'm going to copy what you have for secrets management and stick it in a new PR.